src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java

```diff
@@ -378,13 +404,25 @@ public String getTopic() {
     return topic;
   }
 
+  @SuppressWarnings("deprecated")
   public boolean logErrors() {
-    return !tolerateErrors() || getBoolean(ERRORS_LOG_ENABLE_CONFIG);
```

Comment:
Note: this was added recently from Mongo, which made a difference here 
https://github.com/confluentinc/mongo-kafka/pull/5/files#diff-cd6672a31761782e6b18f3e207148848f7950593821ef36dcad8d1872d570745L302-L307